### DISCUSSION TITLE
Issue Tracker and Project Management Module

### DISCUSSION THREAD

**Discussion by DrewBrasher** [OrchardCMS/OrchardCore#14212] [Ideas]
I would like to build an a open source module that would add Issue Tracker and Project Management features to an Orchard Core CMS site. If anyone would be interested in helping with that or even just providing some guidance or opinions on the design of the module please let me know.

I would like to go ahead and start a git repository for it but I'm still trying to come up with a name for it so if anyone has an idea for a good name for a module like this, I would love to have some suggestions.

**Comment by hishamco** [comment]
I can help with that if it can goes to [OCC](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules) which contains a set of OC community modules

**Reply by DrewBrasher** [reply]
I had thought about putting it there but don't know how to go about getting a new module added to that repo. I'm all for it being part of that.

**Reply by hishamco** [reply]
Please open the issue there, then we will figure out how we can start and more important specify the scope

**Reply by DrewBrasher** [reply]
I've opened an issue: https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/issues/101

**Comment by Piedone** [comment]
Out of curiosity, what's your goal with this module, specifically? I guess you don't want to compete with the likes of Jira, Trello, or GitHub itself.

**Reply by DrewBrasher** [reply]
We have some Help Desk ticket and project management pages on our organization's intranet that need to be updated. They were built years ago in the old asp.net Web Forms framework. I did consider using GitHub, Jira, or something like that instead of rebuilding our own system. One reason I'm leaning toward building our own system is so that our users don't have to create an account on some other website to submit requests.

**Reply by DrewBrasher** [reply]
Our current helpdesk ticket system is used to put in requests to both my area (application development) and IT (computer, printer, network, etc). We also have other departments that want to start using it to keep up with requests and projects they are working on which is part of the reason we need an updated system.

**Reply by Piedone** [reply]
I see, thanks. I obviously don't know your complete use-case but I suggest you check out Jira Service Desk and competitors: there, people just submitting requests (vs the support team) don't need accounts, just send emails. The other accounts you can connect to Azure AD or else.

I'm not necessarily advocating for Jira specifically, just developing such a system seems such a daunting task (though it'd be really cool) that in your place I'd look around more for existing systems.

**Reply by hishamco** [reply]
You're right Zoltan, the aim is to build something minimalistic, for tracking the issue, then it's up to the developer to extend it. I think it's good to utilize the OC features to develop such thing

**Reply by rwxzig** [reply]
This is a really cool idea, I am not very familiar with OC yet, but I will keep an eye on it and see if I can help with something.
